### PR TITLE
Script for Exporting Processed Workflows per Week

### PR DESCRIPTION
**Pull request by KatrinIhler**
With this script, you can export how many workflows ran per week for a given time frame (currently has to be in the same year). If you choose only import workflows, this results in the number of processed event per week. There's also a basic gnuplot script file included to plot the data for those nice graphs management people love. ;)

demo_plot

_This work was sponsored by SWITCH._

### CODE REVIEW

File: export-workflow-statistics/config.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Configuration
+
+url = "https://opencast.com"
+url_pattern = "https://{}.opencast.com"
```
Comment (CGreweling) [inline export-workflow-statistics/config.py:4]: I have set url and url_pattern to the opencast admin url.
This is only neccessarry for non multi tenancy.
If thats correct a hint somewhere would be nice

**Review by CGreweling — COMMENTED** [review]
Works, after your @KatrinIhler  last commit.

File: export-workflow-statistics/config.py (hunk 1 above)
Comment (gregorydlogan) [reply export-workflow-statistics/config.py:4]: +1. And perhaps defaults to localhost? Having the default point to our main website could be confusing.

**Comment by gregorydlogan** [comment]
I've addressed the comments, and pushed this since it's useful.

**Pull request merged**